docs/source/_templates/version-switcher.html

```diff
@@ -0,0 +1,56 @@
+<!-- Copied from https://github.com/pydata/pydata-sphinx-theme/pull/436
+TODO update this once that is included in the upstream theme -->
+
+<div class="dropdown">
+    <button type="button" class="btn btn-primary btn-sm navbar-btn dropdown-toggle" id="switcher_button" data-toggle="dropdown">
+        {{ switcher_prefix }}{{ release }}
```

Comment:
I'm curious: where does `switcher_prefix` come from?

Reply:
Ah, I forgot to check if I could remove it (I asked exactly the same on the upstream PR :))